tests/includeme_test.py

```diff
@@ -43,6 +43,7 @@ def test_bad_schema_not_validated_if_spec_validation_is_disabled(_):
     settings = {
         'pyramid_swagger.schema_directory': 'tests/sample_schemas/bad_app/',
         'pyramid_swagger.enable_swagger_spec_validation': False,
+        'pyramid_swagger.enable_api_doc_views': False,
```

Comment:
`pyramid_swagger.get_swagger_spec` was `mock`'d out in order to avoid testing its details with a bad spec. If you enable the API doc views, it attempts to parse the posix path of spec refs, which ends up trying to compare two `MagicMock` objects. This succeeds in py27, but fails in py35. I assumed this test wasn't trying to catch those types of errors, so I just told it to ignore the api doc routes. Lots of other tests verify those properly anyway.